src/astro/sql/operators/raw_sql.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict
+
+from astro.sql.operators.base import BaseSQLOperator
+
+
+class RawSQLOperator(BaseSQLOperator):
+    """
+    Given a SQL statement, (optional) tables and a (optional) function, execute the SQL statement
+    and apply the function to the results, returning the result of the function.
+
+    Disclaimer: this could potentially trash the XCom Database, depending on the XCom backend used
+    and on the SQL statement/function declared by the user.
+    """
+
+    def execute(self, context: Dict):
```

Comment:
Should we add the return type ?

Reply:
@kaxil I found this one a bit tricky since it can potentially return anything - depending on the function defined by the user. What do you suggest?

Also, I had mixed feelings if we should by default return None or the results. Any feedback is very much appreciated

Reply:
@kaxil I'm merging this, and we can improve it in a next PR

Reply:
How about just adding `-> Any`

Reply:
What if the handler is not provided, @kaxil ? Do you think it would be more intuitive to return None or the result of running the query? I feel the second would be more intuitive for an operator called `run_raw_sql` - IMO, we should not enforce users to define the handler to retrieve results.

The tricky part is that there may be too many results, and that could thrash the database - but it feels more intuitive

Reply:
cc @mikeshwe Something to discuss on tomorrow's call

Reply:
BTW: addressing the `Any` comment in #426